### PR TITLE
UI: Updates the category store based on updated categories api

### PR DESCRIPTION
**Pull request by pratap0007**
# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [tests](https://github.com/tektoncd/community/blob/main/standards.md#principles) (if functionality changed/added)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/main/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/main/CONTRIBUTING.md) for more details._

### CODE REVIEW

File: ui/src/store/resource.test.ts [hunk 1]
```diff
@@ -683,7 +683,7 @@ describe('Store functions', () => {
         store.setURLParams('?category=Automation%2CBuild+Tools&catalog=tekton');
         store.parseUrl();
 
-        expect(store.filteredResources.length).toBe(1);
+        expect(store.filteredResources.length).toBe(7);
```
Comment (PuneetPunamiya) [inline ui/src/store/resource.test.ts:686]: The result will change I guess once we update the logic for filter

**Review by PuneetPunamiya — COMMENTED** [review]
Just had a question, does the UI break on filtering the resources based on categories, since the logic of filter is addressed in another PR, in short what will be the output on UI when any category is selected ??

File: ui/src/store/resource.test.ts (hunk 1 above)
Comment (pratap0007) [reply ui/src/store/resource.test.ts:686]: yes

**Comment by pratap0007** [comment]
> Just had a question, does the UI break on filtering the resources based on categories, since the logic of filter is addressed in another PR, in short what will be the output on UI when any category is selected ??

This pr contains only category store related changes and UI would not break if you select any category

**Comment by PuneetPunamiya** [comment]
> > Just had a question, does the UI break on filtering the resources based on categories, since the logic of filter is addressed in another PR, in short what will be the output on UI when any category is selected ??
> 
> This pr contains only category store related changes and UI would not break if you select any category

What would be the result ??

**Comment by pratap0007** [comment]
> > > Just had a question, does the UI break on filtering the resources based on categories, since the logic of filter is addressed in another PR, in short what will be the output on UI when any category is selected ??
> > 
> > 
> > This pr contains only category store related changes and UI would not break if you select any category
> 
> What would be the result ??

There is no logic to filter resources by categories in this pr ,[ TODO](https://github.com/tektoncd/hub/pull/267/files#diff-bd6a2c3f6618db9b55ed3c7e592499eebbe515d089331fbd7ac729ff5fc4c571R356)

**Comment by PuneetPunamiya** [comment]
Could you please rebase the pr ??

**Comment by PuneetPunamiya** [comment]
Thanks 🤙🏻 
/approve

**Pull request merged**